implants/lib/eldritch/src/random/string_impl.rs

```diff
@@ -0,0 +1,49 @@
+use anyhow::Result;
+use rand::{distributions::Alphanumeric, Rng}; // 0.8
+
+pub fn string(length: u64) -> Result<String> {
+    let s: String = rand::thread_rng()
+        .sample_iter(&Alphanumeric)
```

Comment:
Looks like this might be a cleaner / newer way to write this also:

let string = Alphanumeric.sample_string(&mut rand::thread_rng(), 16);

https://stackoverflow.com/a/72977937

Reply:
I like it! Updated